server/models/users.js

```diff
@@ -0,0 +1,49 @@
+import mongoose from 'mongoose';
+
+const sectionsSchema = mongoose.Schema({
+    sectionName : String,
+    sectionID : Number,
+    sectionHeader : String,
+    sectionContent: {
+        sectionChildName : String,
+        sectionChildImage : String,
+        sectionChildDesc : String
+    }
+})
+
+const clubsSchema = mongoose.Schema({
```

Comment:
don't use words like club as we are making it general
you can use User instead

Reply:
everything else is ok ?

Reply:
Also, use seperate file for both model